artemis-core-client/pom.xml

```diff
@@ -104,6 +104,11 @@
          <artifactId>netty-transport-native-kqueue</artifactId>
          <classifier>${netty-transport-native-kqueue-classifier}</classifier>
       </dependency>
+      <dependency>
+         <groupId>io.netty.incubator</groupId>
+         <artifactId>netty-incubator-transport-native-io_uring</artifactId>
+         <classifier>${netty-transport-native-io_uring-classifier}</classifier>
+      </dependency>
```

Comment:
I think this should be optional for now until close to the point its considered non experimental. I expect few to use it initially, and those who do can easily add it themselves, so it seems premature to make it a hard dependency.

Reply:
Whilst i agree to testing. I disagree with not including the jar. Its behind a feature toggle, by not including it youre making it much tricker for users to test/try and give feedback on. Even myself whilst i can package and deploy to lower envs the apache dist for  better real testing, and getting feedback i have to jump through alot of hoops to the point it makes it prohibitive to do that and get that feedback loop

Reply:
I would imagine it to be the same with softwaremill who run perf tests where they will just take the distro only. Which is one of the main targets we would be asking them to retest with this.

Reply:
I dont consider that an issue at this stage. I think its pretty trivial to customise things like this, little different to getting the distribution if thats what you do (many dont). Certainly not a good reason to include it, for me.